Makefile

```diff
@@ -162,9 +162,8 @@ generate-mocks: install-mock-generators
 	mockery --name '.*' --dir=engine/execution/state --case=underscore --output="./engine/execution/state/mock" --outpkg="mock"
 	mockery --name '.*' --dir=engine/consensus --case=underscore --output="./engine/consensus/mock" --outpkg="mock"
 	mockery --name '.*' --dir=engine/consensus/approvals --case=underscore --output="./engine/consensus/approvals/mock" --outpkg="mock"
-	mockery --name '.*' --dir=fvm --case=underscore --output="./fvm/mock" --outpkg="mock"
+	rm -rf ./fvm/environment/mock
```

Comment:
why do we need this rm only for this directory? doesn't mockery simply overwrite existing files?

Reply:
it doesn't clean up after file renames

Reply:
we should probably wipe every mock directories, but that require moving files first (some mock directories contain non-generated code)